container/templates/build-docker-compose.j2.yml

```diff
@@ -18,5 +18,6 @@ ansible-container:
     - {{ base_path }}:/ansible-container/:Z
     {% if with_volumes %}{% for vol in with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
+    {% if params.playbook %}- {{ params.playbook }}:/ansible-container{{ params.playbook }}:Z {% endif %}
```

Comment:
@chouseknecht yep, checked against playbooks owned by root, it works.

@dustymabe gotcha, `:Z` remains for now.